README.md

```diff
@@ -5,34 +5,39 @@ The first step to get there is to allow charmers to define the status as a pool
 
 # What this is
 
-This charm lib exposes utilities to create 'status pools'. 
+This charm lib exposes utilities to create 'status pools'.
 ```python
-from compound_status import *
-
-class MyPool(StatusPool):
-    """This defines my charm's status pool."""
-    workload = Status()  # this tracks the workload status
-    relation_1 = Status()  # this tracks my integration #1
-    relation_2 = Status()  # this tracks my integration #2
+from compound_status import StatusPool, Status
 
-    
 class MyCharm(CharmBase):
     def __init__(self, framework, key=None):
         super().__init__(framework, key)
-        status_pool = MyPool(self)
+        status_pool = StatusPool(self)
+        status_pool.add(Status("workload"))  # this tracks workload status
+        status_pool.add(Status("relation_1"))  # this tracks my integration #1
+        status_pool.add(Status("relation_2"))  # this tracks my integration #2
+
+        status_pool.set_status("relation_1", ActiveStatus('✅'))
+        status_pool.relation_2 = ActiveStatus('✅')
+
+        workload_status = status_pool.workload
+        workload_status.status = ActiveStatus('✅')
```

Comment:
Seems like trying to accommodate everything and the kitchen sink here is a bit much.  I'd prefer we settle on one canonical way to set/retrieve status.  I'd almost be inclined to just do `status_pool['relation_1'] = ActiveStatus()`